modules/monitor/software/latency_monitor.h

```diff
@@ -0,0 +1,51 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <tuple>
+#include <unordered_map>
+#include <vector>
+
+#include "modules/common/latency_recorder/proto/latency_record.pb.h"
+#include "modules/monitor/common/recurrent_runner.h"
+
+namespace apollo {
+namespace monitor {
+
+using apollo::common::LatencyRecordMap;
+using apollo::common::LatencyReport;
```

Comment:
Avoid making alias in header file. They introduced `apollo::monitor::LatencyReport` into the binary which is confusing and could cause problem.